lib/discourse_api/single_sign_on.rb

```diff
@@ -25,6 +26,36 @@ def self.sso_url
       raise RuntimeError, "sso_url not implemented on class, be sure to set it on instance"
     end
 
```

Comment:
Initialization is similar to `parse`, except we don't need to stringily the keys